deploy/runtools/firesim_topology_with_passes.py

```diff
@@ -226,15 +258,16 @@ def mapping_use_one_f1_16xlarge(self):
 
         for switch in switches:
             self.run_farm.f1_16s[f1_16s_used].add_switch(switch)
-            downlinknodes = map(lambda x: x.get_downlink_side(), switch.downlinks)
-            if all([isinstance(x, FireSimServerNode) for x in downlinknodes]):
+            alldownlinknodes = map(lambda x: x.get_downlink_side(), switch.downlinks)
+            if all([isinstance(x, FireSimServerNode) for x in alldownlinknodes]):
+                downlinknodes = cast(List[FireSimServerNode], alldownlinknodes)
```

Comment:
Again, this is more verbose because `mypy` can't detect that `alldownlinknodes` are of type `FireSimServerNode` from this `if` statement. Maybe this is re-workable but this works for now.